### DISCUSSION TITLE
electron版本调试断点无效问题如何处理

### DISCUSSION THREAD

**Discussion by lusida** [opensumi/core#1818] [Q&A]
ide-electron自带的Debug Main Process调试配置启动断点不生效，请问需要调整什么配置？

**Comment by lusida** [comment]
browser和node两端的源码调试什么配置？

**Reply by erha19** [reply]
看一下这个文档 https://opensumi.com/zh/docs/develop/how-to-contribute#%E8%B0%83%E8%AF%95 @lusida

**Comment by erha19** [comment]
针对 `ide-electron` 项目断点不生效问题，在 VS Code 调试下比较容易出现，可以重新编译项目后直接断点 Main 进程源码试试，该调试配置只针对 main 进程代码，你也可以直接 `yarn start` 启动后，通过声明的  `9229` 端口去直接断点。

参考：https://opensumi.com/zh/docs/develop/how-to-contribute#%E6%8F%92%E4%BB%B6%E8%BF%9B%E7%A8%8B
